variant-errors.go

```diff
@@ -17,3 +17,15 @@ func NewErrVariantNotFound(id int64) ErrVariantNotFound {
 		id,
 	}
 }
+
+// ErrVariantNotFoundByPosition is thrown when no variant is found with the id
+type ErrVariantNotFoundByPosition struct{}
+
+func (err ErrVariantNotFoundByPosition) Error() string {
+	return fmt.Sprintf("variant not found")
+}
+
+// NewErrVariantNotFoundByPosition builds the error
+func NewErrVariantNotFoundByPosition() ErrVariantNotFoundByPosition {
+	return ErrVariantNotFoundByPosition{}
+}
```

Comment:
```suggestion
var ErrVariantNotFoundByPosition = errors.New("variant not found")
```

Reply:
Ha, that's a good point. I am not actually passing anything in.